src/neural/network_blas.cc

```diff
@@ -16,9 +16,10 @@
  along with Leela Chess.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "neural/CL/transforms.h"
-#include "neural/factory.h"
 #include "neural/network.h"
+#include "BLAS/blas.h"
+#include "BLAS/transforms.h"
+#include "neural/factory.h"
```

Comment:
These `neural/` leading parts can be dropped

Reply:
Done, thanks